server/redis.py

```diff
@@ -31,6 +33,23 @@ def append_to_stream(key: str, data: Dict, *args):
         logger.error(f'Error while writing stream to Redis: {e}')
 
 
+def get_stream(key: str, next_item: str = '-'):
+    try:
+        data = redis_data.xrange(key, min=next_item, count=LOG_PAGE_SIZE + 1)
+        if len(data) > LOG_PAGE_SIZE:
+            moves = dict(data[:-1]).values()
+            next_item = data[-1][0]
+            token = sha1(next_item.encode()).hexdigest()
+            save_string(token, next_item)
+        else:
+            moves = dict(data).values()
```

Comment:
This looks awesomw, you can add a `more_results` key (boolean) which can be False when it's the last page. It's not mandatory, but it will make things easier for the clients that use this endpoint